docs/user_manual/boxes.md

```diff
@@ -0,0 +1,328 @@
+option available.
+
+To scan a box, first open the box scanner software on the computer (outside of MISO). Then, go to the Edit Box
+page by clicking the box name in the Boxes list page. The Contents section header has an Options menu on the
+far right of the header; hover over it to reveal the menu, and click "Scan with \<scannerName\>", where
+`\<scannerName\>` is a box scanner that your MISO is configured for. If "Scan with \<scannerName\>" is not visible
+in the list, your box may not be scannable or MISO may not be configured to use this box scanner.
+
+Wait until the Scan dialog box prompts you to place the box on the scanner. It will automatically scan the box. 
+Once the scan is complete, the diagram will display the tubes which the scanner found.
+
+Once a successful scan is made, review the changes that would be made (see figure {{figure}}) and click "Save".
+
+{% include userman-figure.md num=figure cap="Successful box scan" img="box-scan-success.png" %}
+
+If there are any errors with the scan, the scan will need to be retried until there are no errors. Possible
+sources of error are:
+ * The scanning software is not open
+    * This will cause a "could not find the scanner" error message. Open the scanner software on the computer
+      and rescan.
+ * A "No Read" scan result for a tube
+    * This is often caused by dust on the scanner surface or frost on the tubes. Clean the scanner surface
+      and/or tube bottoms and rescan.
+ * A barcode is scanned which is not associated with an item in MISO
+    * Ensure that the tube barcode is correctly associated with an item in MISO.
+ * The network is down or MISO cannot connect to the scanner
+    * Fix these problems first, then rescan.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Exporting Box Information" %}
+
+Go to the Edit Box page by clicking on the box name in the Boxes list page. The Contents section header has an
+Options menu on the far right of the header; hover over it to reveal the menu, and click "Export Box to Excel".
+The Excel spreadsheet containing information about the box contents and their positions will be downloaded.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Filling a Box by Barcode Pattern" %}
+
+Sometimes it may be helpful to assign user-generated barcodes to items (especially if they are not already in
+barcoded tubes), and these user-generated barcodes can be used to quickly put items into boxes.
+
+For example, a plate can be represented in MISO as a non-scannable box. Adding items to this plate in MISO
+allows for easy tracking of which item is in which plate position. Since items in a plate do not have a tube
+barcode it is possible to [add a single item to a plate](#adding_a_single_item_to_a_box) or even to
```

Comment:
"Items in a plate do not have a tube barcode and while it is still possible ... , it can be tedious..."